MDANSE/Src/MDANSE/MolecularDynamics/Trajectory.py

```diff
@@ -548,6 +582,12 @@ def write_atom_properties(
             )
         else:
             type_dataset = self._h5_file["/atom_database/property_types"]
+        if "property_units" not in group:
+            unit_dataset = group.create_dataset(
+                "property_units", data=200 * [""], dtype=string_dt
```

Comment:
Weird thing, but since the final datatype will be a list, I prefer the list first. Feel free to ignore.
```suggestion
                "property_units", data=[""] * 200, dtype=string_dt
```